.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+on:
+  - pull_request
+  - push
+
+name: CI
+
+jobs:
+  tests:
+    name: Tests
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
```

Comment:
We can probably throw in `windows-latest` for a good measure, no?

Reply:
After I run some PHP versions on `windlows-latest` during GitHub Action, I suffer running has been done with exited code 1.

And the [GitHub Action log](https://github.com/open-source-contributions/array_key_first_last-polyfill/actions/runs/221667188) is available here on my forked repository.